tests/test.rb

```diff
@@ -2,8 +2,9 @@
 require 'safe_yaml'
 
 data = SafeYAML.load_file '_data/project_filter.yml', safe: true
-expect_size = data.size
-projectdir = Dir.new('_site/dashboard/project')
+redirects = Dir.open('pages/project').count-2
```

Comment:
the `count-2` excludes ignores `.` and `..` so any real files in that directory would be get counted. TBH these aren't great tests to begin with. They only tell us how many pages were generated but nothing about whether they're correct. I'd rather put more energy into #217 to get better tests in place and then deprecate this one.

I do like the idea of generating redirects out of a data file instead of making more of these files, though.